lib/team_result_parser.rb

```diff
@@ -96,28 +96,18 @@ def get_win_percentage(season_games)
         end
       end
     end
-    # Add default value for hash (Hash.new(0))
-    coach_played = {}
+    coach_played = Hash.new(0)
     games_played.each do |play|
-      if coach_played[play.head_coach] == nil
-        coach_played[play.head_coach] = 1
-      else
-        coach_played[play.head_coach] += 1
-      end
+      coach_played[play.head_coach] += 1
     end
     win_games = games_played.find_all do |w_game|
       w_game.result == "WIN"
     end
-    coach_win = {}
+    coach_win = Hash.new(0)
```

Comment:
Great idea on cutting some lines by calling `Hash.new` here and defaulting it's value to `0`